applications/serial_lte_modem/src/slm_sockopt.h

```diff
@@ -29,7 +29,8 @@ enum at_sockopt {
 	AT_SO_RAI_ONE_RESP = 52,
```

Comment:
Libmodem changes crash the DK with SLM tests and fail several test sets. Investigation ongoing.